.eslintrc.js

```diff
@@ -1,11 +1,22 @@
 module.exports = {
   extends: ['algolia', 'algolia/react'],
   rules: {
-    'react/prop-types': 0,
-    'no-param-reassign': 0,
     'eslint-comments/disable-enable-pair': 0,
+    // Do not enforce a specific import order
+    'import/order': 0,
+    // Allow function components
+    'react/function-component-definition': 0,
```

Comment:
I think we should enable this rule so the React DevTools show the name of each component in the tree vs. a bunch of anonymous components. wdyt?

This can be done in a separate PR.